paper-server/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java

```diff
@@ -83,12 +136,72 @@ private static Component getUpdateStatusMessage(final String repo, final ServerB
         };
     }
 
+    private static @Nullable String fetchMinecraftVersionList(final ServerBuildInfo build) {
+        final String currentVersion = build.minecraftVersionId();
+        final String userAgent = "Paper/" + currentVersion + " (https://papermc.io/)";
```

Comment:
In the user agents, can you please use VERSION_SIMPLE rather than just minecraft version?
Additionally, use the brand name rather than hard coded "Paper"